### PR TITLE
Tajo jdbc thrift interpreters

### PR DESCRIPTION
**Pull request by khalidhuseynov**
This is for decoupling Tajo-jdbc and Tajo-thrift interpreters. 

TODO
- [x] Rename Tajo interpreter explicitly into tajo-jdbc
- [x] Add tajo-thrift interpreter
- [x] Implement **getProgress()** for tajo-thrift interpreter, by @bzz 
- [x] Learning Zeppelin architecture and handling unexpected problems 
- [ ] Implement **cancel()**

### CODE REVIEW

File: tajo-thrift/pom.xml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>zeppelin</artifactId>
+    <groupId>org.apache.zeppelin</groupId>
+    <version>0.5.0-SNAPSHOT</version>
```
Comment (bzz) [inline tajo-thrift/pom.xml]: this is old version number, after ZEPPELIN-65 AKA #59 and after ASF incubation policy version identifier should include -incubating and now looks like '0.5.0-incubating-SNAPSHOT'

**Comment by bzz** [comment]
@khalidhuseynov build fails, can you fix it please?

**Comment by khalidhuseynov** [comment]
Triggering CI Build

**Comment by bzz** [comment]
Shall we update or close this one, after #39 ?

**Pull request closed without merging**